addons/hr_attendance/security/hr_attendance_security.xml

```diff
@@ -4,82 +4,62 @@
-        <field name="implied_ids" eval="[(4, ref('hr.group_hr_user')), (4, ref('hr_attendance.group_hr_attendance'))]"/>
+    <record id="group_hr_attendance_officer" model="res.groups">
+        <field name="name">Officer: Manage attendances</field>
+        <field name="category_id" ref="base.module_category_hidden"/>
+        <field name="comment">The user will have access to the attendance records and reporting of employees where he's set as an attendance manager</field>
     </record>
 
     <record id="group_hr_attendance_manager" model="res.groups">
         <field name="name">Administrator</field>
         <field name="category_id" ref="base.module_category_human_resources_attendances"/>
-        <field name="implied_ids" eval="[(4, ref('hr_attendance.group_hr_attendance_user'))]"/>
         <field name="users" eval="[(4, ref('base.user_root')), (4, ref('base.user_admin'))]"/>
+        <field name="implied_ids" eval="[(4, ref('hr_attendance.group_hr_attendance_officer'))]"/>
     </record>
 
     <record id="base.default_user" model="res.users">
         <field name="groups_id" eval="[(4,ref('hr_attendance.group_hr_attendance_manager'))]"/>
     </record>
 
-    <record id="group_hr_attendance_use_pin" model="res.groups">
-        <field name="name">Enable PIN use</field>
-        <field name="category_id" ref="base.module_category_hidden"/>
-        <field name="comment">The user will have to enter his PIN to check in and out manually at the company screen.</field>
-    </record>
-
     <data noupdate="1">
 
+        <!-- Attendances -->
         <record id="hr_attendance_rule_employee_company" model="ir.rule">
             <field name="name">Employee multi company rule</field>
             <field name="model_id" ref="model_hr_attendance"/>
             <field name="global" eval="True"/>
             <field name="domain_force">['|',('employee_id.company_id','=',False),('employee_id.company_id', 'in', company_ids)]</field>
         </record>
 
-        <record id="hr_attendance_rule_attendance_manager" model="ir.rule">
-            <field name="name">attendance officer: full access</field>
+        <record id="hr_attendance_rule_attendance_simple_user" model="ir.rule">
+            <field name="name">Attendance base user: Read his own attendances</field>
             <field name="model_id" ref="model_hr_attendance"/>
-            <field name="domain_force">[(1,'=',1)]</field>
-            <field name="groups" eval="[(4,ref('hr_attendance.group_hr_attendance_user'))]"/>
+            <field name="domain_force">[('employee_id.user_id', '=', user.id)]</field>
+            <field name="groups" eval="[(4,ref('hr_attendance.group_hr_attendance_own_reader'))]"/>
         </record>
 
-        <record id="hr_attendance_rule_attendance_manual" model="ir.rule">
-            <field name="name">Manual Attendance: own attendances</field>
+        <record id="hr_attendance_rule_attendance_admin" model="ir.rule">
```

Comment:
Could be placed after hr_attendance_rule_attendance_manager_restrict as the ir.rules seems to increase in access levels.